src/main/webapp/index.jsp

```diff
@@ -38,7 +38,8 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+      <h2>The Avengers: CodeU Spring 2018 Team 27</h3>
```

Comment:
opening tag is h2, but closing tag is h3

Reply:
@krugg-codeu updated! thank you